paging_test.py

```diff
@@ -3423,19 +3423,31 @@ def test_failure_threshold_deletions(self):
         supports_v5_protocol = self.supports_v5_protocol(self.cluster.version())
 
         self.fixture_dtest_setup.allow_log_errors = True
+        supports_guardrails = self.cluster.version() >= LooseVersion('4.0')
+        if supports_guardrails:
+            config_opts = {'guardrails': {'tombstone_failure_threshold': 500,
+                                          'tombstone_warn_threshold': -1,
+                                          'write_consistency_levels_disallowed': {}},
+                           'tombstone_failure_threshold': 0,
```

Comment:
that looks dangerous. why the "same" setting  in two different places with different values?

Reply:
I spoke to Aleksandr about this; it's likely that at the time the original test was added, `tombstone_[faulure|warn]_threshold` entries were still in `cassandra.yaml` as top-level properties with non-zero defaults. Using this would set those values to zero, which then prevents the guardrail settings from being overwritten.

Since then, `tombstone_[failure|warn]_threshold` have been removed from `cassandra.yaml` so this isn't a concern in the test anymore. I've removed these extra settings and the tests pass.